src/ORM/FieldType/DBString.php

```diff
@@ -30,8 +30,13 @@ abstract class DBString extends DBField
      */
     public function __construct($name = null, $options = [])
     {
-        $this->options['nullifyEmpty'] = true;
-        $this->options['default'] = '';
+        $options = is_array($options) ? $options : [];
+        if (!array_key_exists('nullifyEmpty', $options)) {
+            $options['nullifyEmpty'] = true;
+        }
+        if (!array_key_exists('default', $options)) {
+            $options['default'] = '';
+        }
```

Comment:
Only set these default options if explicit option values for these keys aren't being passed in